src/webots/nodes/WbMesh.cpp

```diff
@@ -157,12 +158,47 @@ void WbMesh::updateTriangleMesh(bool issueWarnings) {
     return;
   }
 
+  if (mMaterialIndex->value() >= (int)scene->mNumMaterials) {
+    warn(tr("Geometry with color index \"%1\" doesn't exist in the mesh.").arg(mMaterialIndex->value()));
+    return;
+  }
+
+  // Assimp fix for up_axis
+  // Adapted from https://github.com/assimp/assimp/issues/849
+  int UpAxis = 1, UpAxisSign = 1, FrontAxis = 2, FrontAxisSign = 1, CoordAxis = 0, CoordAxisSign = 1;
+  double UnitScaleFactor = 1.0;
```

Comment:
Oops, I missed this minor coding style issue: variables should be camelCase, i.e., start with a lowercase letter.

Reply:
Thank you I will do this.

Reply:
Fixed in https://github.com/cyberbotics/webots/pull/4193.